go.mod

```diff
@@ -58,7 +58,7 @@ require (
 	kubevirt.io/api v0.0.0-20211117075245-c94ce62baf5a
 	kubevirt.io/containerized-data-importer-api v1.41.0
 	sigs.k8s.io/apiserver-network-proxy v0.0.24
-	sigs.k8s.io/cluster-api v1.1.3
+	sigs.k8s.io/cluster-api v1.1.4
 	sigs.k8s.io/cluster-api-provider-aws v1.1.0
 	sigs.k8s.io/cluster-api-provider-azure v1.1.1
 	sigs.k8s.io/cluster-api-provider-ibmcloud v0.2.0
```

Comment:
and also is there a plan for updating `sigs.k8s.io/cluster-api-provider-ibmcloud` to latest available package and the image?

Reply:
Updated.